app/styles/modal-dialog.less

```diff
@@ -0,0 +1,65 @@
+.oss-modal-dialog-backdrop {
+  background-color: var(--color-backdrop);
+  height: 100vh;
+  width: 100vw;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1024;
+}
+
+.oss-modal-dialog-sm {
+  width: 600px;
+}
+
+.oss-modal-dialog-md {
+  width: 700px;
+}
+
+.show-modal {
+  animation: fadeInAnimation 0.3s ease-in-out;
+}
+
+.oss-modal-dialog {
```

Comment:
if you use the `header/footer` tags, why not skip the classes usage as it's unique enough